docs/content/docs/user-docs/leader-election.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "Leader election"
+description: "Configure leader election for ACK controllers"
+lead: "How to enable leader election for ACK controllers"
+draft: false
+menu:
+  docs:
+    parent: "getting-started"
+weight: 60
+toc: true
+---
+
+In a Kubernetes cluster, multiple instances of a controller maybe running
+simultaneously to manage resources and perform tasks. However, to avoid conflicts
```

Comment:
```suggestion
In a Kubernetes cluster, you may want to run multiple instances of any ACK controller - configured for different accounts or regions, or for fail state rollover. However, to avoid conflicts
```